scripts/test-codingstyle-py.sh

```diff
@@ -8,11 +8,21 @@ FILES=$(find . -iname "*.py" -not -name "cpplint.py" -not -path "./doc/*")
 FILES+=" "
 FILES+=$(grep -rl "^#./usr/bin.*python" ./* | grep -v cpplint.py | grep -vE "^\.\/doc\/")
 
+if [ "$TRAVIS_COMMIT_RANGE" != "" ]; then
+    CHANGED_FILES=$(git diff --name-only $TRAVIS_COMMIT_RANGE)
+    printf "Looking at changed files:\n${CHANGED_FILES}"
+    FILES=$(comm -12 <(sort <(for f in $FILES; do printf "$f\n"; done)) <(sort <(for f in $CHANGED_FILES; do printf "./$f\n"; done)))
```

Comment:
Say what?

Reply:
Use `git diff` to get a list of files changed in this push
Use `comm` to compare two files
* `<( _expression_)` will generate a temporary file
* Sort and Iterate over both lists and add a newline character